src/algebra/monoid_algebra/degree.lean

```diff
@@ -32,11 +33,12 @@ variables [add_monoid A] [covariant_class A A (+) (≤)] [covariant_class A A (f
 lemma degree_mul_le (f g : add_monoid_algebra R A) :
   (f * g).degree ≤ f.degree + g.degree :=
 begin
-  refine (finset.fold_max_le _).mpr ⟨bot_le, λ d ds, _⟩,
+  refine (finset.fold_sup_le _).mpr ⟨bot_le, λ d ds, _⟩,
```

Comment:
Can't you use `finset.sup_le` here instead?

Reply:
Yes, `finset.sup_le` works better!